packages/cli/src/commands/list.ts

```diff
@@ -230,23 +238,21 @@ export default async function main(client) {
               getProjectName(dep),
               chalk.bold((includeScheme ? 'https://' : '') + dep.url),
               stateString(dep.state),
-              chalk.gray(ms(Date.now() - new Date(dep.createdAt))),
+              chalk.gray(ms(Date.now() - dep.createdAt)),
               dep.creator.username,
             ],
           ])
           // flatten since the previous step returns a nested
           // array of the deployment and (optionally) its instances
-          .reduce((ac, c) => ac.concat(c), [])
-          .filter(
-            app == null
-              ? // if an app wasn't supplied to filter by,
-                // we only want to render one deployment per app
-                filterUniqueApps()
-              : () => true
+          .flat()
+          .filter(app =>
+            // if an app wasn't supplied to filter by,
+            // we only want to render one deployment per app
+            app === null ? filterUniqueApps() : () => true
           ),
       ],
       {
-        align: ['l', 'l', 'r', 'l', 'b'],
+        align: ['l', 'l', 'r', 'l'],
```

Comment:
Removed the `b` element of the array. I check and this not affect to align of the table. Also, I was looking for, but there is no `b` in the alignment parameters.

 [text-table API](https://github.com/substack/text-table)

Reply:
Considering that there's 5 columns in the table, I think we should use `l` or `r` as the value, instead of removing the `b` completely.

Reply:
I seem a good idea. Thank You, Nate 👍🏼